src/components/AvatarRow/AvatarRow.utils.js

```diff
@@ -0,0 +1,30 @@
+export const MARGIN_LEFT = 2
+
+export function splitAvatarsArray(avatars, itemsDisplayed) {
+  const hiddenAvatars =
+    itemsDisplayed < avatars.length
+      ? avatars.slice(itemsDisplayed - 1, avatars.length)
+      : []
+
+  const shownAvatars = hiddenAvatars.length
+    ? avatars.slice(0, itemsDisplayed - 1)
+    : avatars
+
+  return { shownAvatars, hiddenAvatars }
+}
+
+export function setupObserver(callback) {
```

Comment:
is that the kind of utils we would like to make it more global so we could _potentially_ reused it in other component?

Reply:
Yes I think so! `ResizeObserver` is pretty cool and useful, I'm going to make a comment to move this to the long coming "utils" revamp that I hope I can tackle soon 😅

Reply:
(the only drawback is that is not that easy to test)